hasura/metadata/databases/databases.yaml

```diff
@@ -6,4 +6,4 @@
         from_env: HASURA_GRAPHQL_DATABASE_URL
       isolation_level: read-committed
       use_prepared_statements: false
-  tables: '!include default/tables/tables.yaml'
+  tables: "!include default/tables/tables.yaml"
```

Comment:
@crabsinger whereas this double quote is Hasura editing metadata files when something is saved from the console.